demos/appyx-interactions/ios/src/iosMain/kotlin/theme/Colors.kt

```diff
@@ -0,0 +1,35 @@
+package theme
+
+import androidx.compose.material.darkColors
+import androidx.compose.ui.graphics.Color
+
+val appyx_yellow1 = Color(0xFFFFC629)
+val appyx_yellow2 = Color(0xFFFFE54A)
+val appyx_dark = Color(0xFF1F2126)
+val appyx_bright = Color(0xFFFFFFFF)
+val color_primary = Color(0xFFFFC629)
+
+val md_indigo_500 = Color(0xFF3F51B5)
+val md_blue_500 = Color(0xFF2196F3)
+val md_light_blue_500 = Color(0xFF03A9F4)
+val md_amber_500 = Color(0xFFFFC107)
+val md_grey_500 = Color(0xFF9E9E9E)
+val md_blue_grey_500 = Color(0xFF607D8B)
+val md_cyan_500 = Color(0xFF00BCD4)
+val md_teal_500 = Color(0xFF009688)
+val md_light_green_500 = Color(0xFF8BC34A)
+val md_lime_500 = Color(0xFFCDDC39)
+val md_pink_500 = Color(0xFFE91E63)
```

Comment:
I'd rather we don't copy these but have them defined in a common module.  It's fine if we do it in a separate PR though.

Reply:
See https://github.com/bumble-tech/appyx/pull/601#discussion_r1335682325